Automattic/Automattic-Tracks-iOS
# Issue/52 use ephemeral session

Closes #52

How to test:
 - Start the demo app
 - Send events and check if they get to the server correctly.

Needs Review: @astralbodies
